warehouse/templates/error-base.html

```diff
@@ -18,7 +18,9 @@
 {% block body %}
 <div class="viewport-section viewport-section--dark">
   <div class="viewport-section__content">
-    <img alt="{{ request.registry.settings['site.name'] }}" src="{{ request.static_path('warehouse:static/dist/images/logo-small.svg') }}">
+    <a href="/">
```

Comment:
Should this be `href="{{ request.route_path('index') }}` instead?

Reply:
yes, will update :)